### PR TITLE
Interactive class

### PR DESCRIPTION
**Pull request by giovp**

### CODE REVIEW

File: src/napari_spatialdata/_interactive.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+class Interactive:
+    def __init__(self, sdata: SpatialData):
+        self._viewer = napari.Viewer()
+        self._sdata = sdata
+        self._sdata_widget = SdataWidget(self._viewer, sdata)
+        self._list_widget = self._viewer.window.add_dock_widget(
+            self._sdata_widget, name="Select spatialdata segment", area="left"
+        )
+        # TODO add to window
+        napari.run()
```
Comment (giovp) [inline src/napari_spatialdata/_interactive.py:83]: add this to separate method e.g. `run()`

**Comment by giovp** [comment]
- [ ] separate spatiadata-widget across elements (one for images, one for labels etc.)
- [ ] updating layer in place (separate PR)
- [ ] remove layer selection qbox from View widget and update view widget directly from active napari layer
- [ ] remove `library_id` from `Model`
- [ ] for now, set scalefactors etc. to default values, and then we'll use the one from spatialdata
- [ ] add spatialdata-widget to window menu bar

**Comment by rahulbshrestha** [comment]
@giovp I think this PR is pretty much done other than the "interactive tests with new layer selection" and "update layer in-place" which will be tracked in https://github.com/scverse/napari-spatialdata/pull/44.

**Comment by rahulbshrestha** [comment]
Changes in this PR have been implemented in https://github.com/scverse/napari-spatialdata/pull/46. Closing this for now.

**Pull request closed without merging**